portfolio/src/main/webapp/css/master.css

```diff
@@ -3,6 +3,23 @@
   overflow-y: auto;
 }
 
+.notification {
+  opacity: 80%;
+  -moz-transition: opacity 1s linear, ;
```

Comment:
remove `, `

And can we put all the `*-transition` attributes under `transition`, and alphabetize them?

Reply:
I think I took the alphabetization from the style guide too literally haha. Looking again at the style guide your suggestion makes sense. Changing now.